user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Ad_hoc_data/Ad_hoc_Tutorials_GQIDMS.md

```diff
@@ -136,3 +136,10 @@ Transform the responses into a [GQIPage](xref:GQI_GQIPage). Each `LoginInfoRespo
 
 > [!NOTE]
 > Ensure `DateTime` values are in UTC. Although the `ConnectTime` value is in UTC, it is not explicitly marked as such. Therefore, this is specified in the example.
+
+> [!NOTE]
+> Don't forget to compile script as [library](xref:Compiling_a_CSharp_code_block_as_a_library) by including following in the Script.Exe tag.
```

Comment:
If this is something the user definitely always has to do, it would be better to add it as another step in the procedure instead of as a note.

Reply:
It is something user always has to do when working with GQI, it's prerequisite for creating GQI DS. Depending on persons experience it is either just a reminder or additional step to be performed. Your call, if you prefer to have it as extra step I can do that, if you feel like this is fine we can leave it as is.

Reply:
As this tutorial is aimed at learners who may not have much experience with GQI, I think adding it as a step would be better than as a reminder.